CIP-XXXX/README.md

```diff
@@ -0,0 +1,447 @@
+        [
+          "witness may be broken up into a sub-array of 64-character strings",
+          "assuming that the overall length of the witness is longer than the",
+          "64-character on-chain metadata limit for strings."
+        ]
+      ],
+      [
+        "26bacc7b88e2b40701387c521cd0c50d5c0cfa4c6c6d7f0901395757",
+        [
+          "this is the second witness for a 2 out of 3 multi-sig script.",
+          "it's also broken up into an array.",
+          "strings in the witness array should be concatenated as-is",
+          "withouts spaces or any formatting applied"
+        ]
+      ]
+    ]
+  }
+}
+```
+
+### Example FT Token Registration Metadata
+
+```json
+{
+  "867": {
+    "1": {
+      "1": "00000002df633853f6a47465c9496721d2d5b1291b8398016c0e87ae",
+      "2": [
+        26
+      ],
+      "3": [
+        "Ed25519"
+      ],
+      "4": 12345,
+      "5": "https://oracle.tokenproject.io/",
+      "6": {
+        "26": {
+          "1": [
+            {
+              "0": [
+                "d894897411707efa755a76deb66d26dfd50593f2e70863e1661e98a0",
+                "7370616365636f696e73"
+              ],
+              "1": "spacecoins",
+              "2": "SPACE",
+              "3": [
+                "the OG Cardano community token",
+                "-",
+                "whatever you do, your did it!",
+                "",
+                "Learn more at https://spacecoins.io!"
+              ],
+              "4": 0,
+              "5": [
+                "https://",
+                "spacecoins.io"
+              ],
+              "6": [
+                "ipfs://",
+                "bafkreib3e5u4am2btduu5s76rdznmqgmmrd4l6xf2vpi4vzldxe25fqapy"
+              ],
+              "7": [
+                [
+                  "ipfs://",
+                  "bafkreibva6x6dwxqksmnozg44vpixja6jlhm2w7ueydkyk4lpxbowdbqly"
+                ],
+                "3507afe1daf05498d764dce55e8ba41e4acecd5bf42606ac2b8b7dc2eb0c305e"
+              ],
+              "8": [
+                "37da68d092f4e61feb237de5e86b404171b59d3880d340023a16a6983491736d",
+                "0"
+              ]
+            }
+          ]
+        }
+      }
+    },
+    "2": [
+      [
+        "e97316c52c85eab276fd40feacf78bc5eff74e225e744567140070c3j",
+        [
+          "witness may be broken up into a sub-array of 64-character strings",
+          "assuming that the overall length of the witness is longer than the",
+          "64-character on-chain metadata limit for strings."
+        ]
+      ]
+    ]
+  }
+}
+```
+
+## Rationale
+
+For this specification, I have drawn inspiration from 
+[CIP-36: Catalyst/Voltaire Registration Metadata Format](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0036)
+which succinctly and canonically publishes data to the main chain (L1) via a metadata transaction and without any
+required modification or customization to the underlying ledger with regard to Native Assets.
```

Comment:
CIP-0036 does however tie together some stake credentials with some voting key. The stake credentials act in this scenario as strawman identity. My major question regarding CIP-0088 is how to prevent spoofing and how the witness is actually connected back to the original metadata owner (which is not necessarily the same entity as the one minting the tokens!).

Reply:
The goal here would be to provide additional context and information (metadata) to the token policy itself. So, I would argue, the holder of the "keys" (or entity responsible for deploying the smart contract) ostensibly does own this particular metadata while things like the metadata attached to a specific token may fall under the "ownership" of an individual holder of that token.

The primary goals being to increase and decentralize discoverability of token projects with centralized ownership and reduce the need for redundant metadata (ledger bloat) information commonly found in currently published metadata standards due to a lack of an over-arching or top-level standard.

In this case, I argue that the signing keys of the Native Script policy constitute sufficient proof of ownership of the project to be "trusted" until such time as a better standard for signaling and/or transferring ownership and control of a native asset policy exists.